ProjectSettings/ProjectSettings.asset

```diff
@@ -266,7 +266,49 @@ PlayerSettings:
   AndroidMinifyDebug: 0
   AndroidValidateAppBundleSize: 1
   AndroidAppBundleSizeToValidate: 150
-  m_BuildTargetIcons: []
+  m_BuildTargetIcons:
+  - m_BuildTarget: Standalone
+    m_Icons:
+    - serializedVersion: 2
+      m_Icon: {fileID: 2800000, guid: cf2b1b1c158010bfb8fb3e8d0275aa8a, type: 3}
+      m_Width: 1024
+      m_Height: 1024
+      m_Kind: 0
+    - serializedVersion: 2
+      m_Icon: {fileID: 0}
```

Comment:
These all have fileID 0, I presume that this is correct.

Reply:
Unity autogenerated that, so I assume yes. Will find that out soon when luuk starts testing builds.